include/deal.II/physics/transformations.h

```diff
@@ -789,6 +857,55 @@ namespace internal
 
 
 
+template<typename Number>
+Tensor<2,2,Number>
+Physics::Transformations::Angular::rotation_matrix_2d (const Number &angle)
+{
+  return Tensor<2,2> (
+  {
+    {
```

Comment:
Yes, let the compiler take care of moving data around as necessary. At least in optimized mode, the end result will be exactly the same.